policyengine_us/parameters/gov/states/ga/tax/income/deductions/standard/base_amount.yaml

```diff
@@ -0,0 +1,30 @@
+description: Georgia provides filers a standard deduction of this amount, depending on filing status.
+metadata:
+  label: Georgia standard deduction
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference: 
+  - title: 2021 IT-511 Individual Income Tax Booklet - page 11
+    href: https://dor.georgia.gov/document/booklet/2021-it-511-individual-income-tax-booklet/download
+  - title: 2022 IT-511 Individual Income Tax Booklet - page 11
+    href: https://dor.georgia.gov/document/document/2022-it-511-individual-income-tax-booklet/download
+  - title: Official Code of Georgia Annotated | CH17 Income Taxes | O.C.G.A. § 48-7-27
```

Comment:
need exact subsection of the law where the number can be found in all yaml files